.github/actions/run-fixtures-test/action.yml

```diff
@@ -0,0 +1,52 @@
+name: Run a fixtures test
+description: Run a fixtures test
+
+inputs:
+  fixtures-dir:
+    description: Path to the fixtures directory
+    required: true
+  command:
+    description: The git-cliff command to run
+    required: false
+    default: ""
+
+runs:
+  using: composite
+  steps:
+    - name: Install toolchain
+      uses: actions-rs/toolchain@v1
+      with:
+        profile: minimal
+        toolchain: nightly
+        override: true
+
+    - name: Install git-cliff
+      run: |
+        cargo install --path git-cliff/
```

Comment:
nitpick: can we make this command one-line?

Reply:
7c9616c70a817a8af922717134c31e7df3e50e08